cmd/cmd.go

```diff
@@ -664,6 +664,8 @@ func generateInteractive(cmd *cobra.Command, opts generateOptions) error {
 		fmt.Fprintln(os.Stderr, "  /set noformat          Disable formatting")
 		fmt.Fprintln(os.Stderr, "  /set verbose           Show LLM stats")
 		fmt.Fprintln(os.Stderr, "  /set quiet             Disable LLM stats")
+		fmt.Fprintln(os.Stderr, "  /set cache             Enable caching")
```

Comment:
I'd like to just enable this by default, so let's not add the CLI set commands in that case.